### PR TITLE
Fix/folder model

### PR DESCRIPTION
**Pull request by kasvith**
Pull Request for Issue #265  .

### Summary of Changes
- Added missed variables

### Testing Instructions
- Create folder model now should appear

### CODE REVIEW

File: administrator/components/com_media/package.json [hunk 1]
```diff
@@ -10,7 +10,9 @@
     "build-js": "browserify ./resources/scripts/mediamanager.js > ../../../media/com_media/js/mediamanager.js"
   },
   "dependencies": {
+    "amdefine": "^1.0.1",
     "babel-polyfill": "^6.23.0",
+    "has-flag": "^2.0.0",
```
Comment (dneukirchen) [inline administrator/components/com_media/package.json]: Please do not add 3dp deps.

File: administrator/components/com_media/resources/scripts/store/state.js [hunk 2]
```diff
@@ -23,6 +23,6 @@ export default {
     selectedDirectory: options.providers[0].name + ':/',
     // The currently selected items
     selectedItems: [],
-    // The currently selected items
-    selectedItems: [],
+    // The state of cre
```
Comment (dneukirchen) [inline administrator/components/com_media/resources/scripts/store/state.js]: cre....? ;-)

Comment (kasvith) [reply administrator/components/com_media/resources/scripts/store/state.js]: ;) was on train at that time 🤓

File: administrator/components/com_media/resources/scripts/store/state.js [hunk 3]
```diff
@@ -23,6 +23,6 @@ export default {
     selectedDirectory: options.providers[0].name + ':/',
     // The currently selected items
     selectedItems: [],
-    // The currently selected items
-    selectedItems: [],
+    // The state of create folder model
+    showCreateFolderModal: false
 }
```
Comment (laoneo) [inline administrator/components/com_media/resources/scripts/store/state.js]: Can you add here the missing empty line as well?

Comment (kasvith) [reply administrator/components/com_media/resources/scripts/store/state.js]: you mean an empty line before the comment?

Comment (kasvith) [reply administrator/components/com_media/resources/scripts/store/state.js]: the original file did not contain any new lines before a comment

Comment (laoneo) [reply administrator/components/com_media/resources/scripts/store/state.js]: Every file should end with an empty line. As you modified it, I tough would be nice to fix that CS issue as well.

**Review by dneukirchen — APPROVED** [review]

**Comment by dneukirchen** [comment]
thx @kasvith

**Pull request merged**